catalog/transformer/catalog-transformer-generic/src/main/java/ddf/catalog/transformer/generic/xml/SaxEventHandlerFactory.java

```diff
@@ -0,0 +1,9 @@
+package ddf.catalog.transformer.generic.xml;
+
+import ddf.catalog.util.Describable;
+
+public interface SaxEventHandlerFactory extends Describable {
```

Comment:
Yeah. All the concrete classes (and the blueprint, natch) moved to that new one with the api definitions (the interfaces) left in this one.